### PR TITLE
issue #3619: Missing moderator ids on forum move

### PR DESCRIPTION
**Pull request by SimSync**
In case the post id is not set use the thread id to look for the moderator ids

### CODE REVIEW

**Comment by phibel** [comment]
The solution looks good to me, but a LAN for new the error message should be added before the merge.

File: e107_plugins/forum/forum_post.php [hunk 1]
```diff
@@ -802,8 +812,17 @@ private function forumSelect($name, $curVal=null, $opts=null)
...
+		{
+			// Forum just moved. No need to display the forum move form again.
+			return;
+		}
+
 		if(!deftrue('MODERATOR'))
 		{
+			// todo: LAN for new error message
+			$mes = e107::getMessage();
+			echo $mes->addWarning('You do not have access to this area!')->render();
```
Comment (Moc) [inline e107_plugins/forum/forum_post.php]: Please use `LAN_NO_PERMISSIONS` (e107_languages/English/English.php

```suggestion
			echo $mes->addWarning(LAN_NO_PERMISSIONS)->render();
```

File: e107_plugins/forum/forum_post.php [hunk 2]
```diff
@@ -802,8 +812,17 @@ private function forumSelect($name, $curVal=null, $opts=null)
...
 	{
+		if (isset($_POST['forum_move']))
+		{
+			// Forum just moved. No need to display the forum move form again.
+			return;
+		}
+
 		if(!deftrue('MODERATOR'))
 		{
+			// todo: LAN for new error message
```
Comment (Moc) [inline e107_plugins/forum/forum_post.php]: can be removed after updating 
```suggestion
```

**Comment by Moc** [comment]
@SimSync I've suggested some changes, please adjust and I can merge it

**Comment by SimSync** [comment]
@Moc Just implemented the changes

**Comment by Moc** [comment]
@SimSync Thanks!! :)

**Pull request merged**